cloudinit/config/schemas/schema-cloud-config-v1.json

```diff
@@ -35,6 +35,8 @@
         "lxd",
         "mcollective",
         "mounts",
+        "netplan-nm-patch",
```

Comment:
We only need the `_` version of all the keys. The `-` versions exist in other modules for backwards compatibility only.

Reply:
Ok